provdebug/Serializer.py

```diff
@@ -0,0 +1,27 @@
+import platform
+import json
+import random
+from .DebugRecord import DebugRecord
+from typing import List, Dict
+
+class Serializer:
```

Comment:
Extracted a new class that is responsible for saving to disk, instead of doing it in `pvdebug.py`